ctapipe_io_lst/__init__.py

```diff
@@ -617,44 +608,69 @@ def fill_pointing_info(self, array_event):
                 'No drive report specified, pointing info will not be filled'
             )
 
-    def fill_r0_camera_container(self, r0_container, zfits_event):
+    def fill_r0r1_camera_container(self, zfits_event):
         """
-        Fill with R0CameraContainer
+        Fill the r0 or r1 container, depending on whether gain
+        selection has already happened (r0) or not (r1)
```

Comment:
isn't it the other way around?
the gain_selected data are written in r1

Reply:
yes, good catch.